### DISCUSSION TITLE
Using the RDKit-Molhash integration on command line

### DISCUSSION THREAD

**Discussion by vfscalfani** [rdkit/rdkit#4164] [Q&A] [answered]
The original NextMove software MolHash code had a command line interface. I don't believe that is part of RDKit, and the RDKit code has incorporated a few MolHash improvements.

My question:

If I want to use the MolHash RDKit integration from the command line, is the easiest way to just create a Python script using argparse? Similar to something like this for MolVS:

https://github.com/rdkit/rdkit/blob/master/Contrib/MolVS/molvs_cli.py

I can do that, but wanted to make sure I am not missing something obvious. 

Thanks,

Vin

**Comment by greglandrum** [comment, marked as answer]
Hi @vfscalfani , yeah I think creating something new based on argparse would be the way to do it
